src/assets/wise5/authoringTool/display-branch-path-steps/display-branch-path-steps.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, Input, SimpleChanges } from '@angular/core';
+import { TeacherProjectService } from '../../services/teacherProjectService';
+import { CommonModule } from '@angular/common';
+
+@Component({
+  imports: [CommonModule],
+  selector: 'display-branch-path-steps',
+  standalone: true,
+  styleUrl: './display-branch-path-steps.component.scss',
+  templateUrl: './display-branch-path-steps.component.html'
+})
+export class DisplayBranchPathStepsComponent {
+  @Input() branchPathNodes: any[] = [];
```

Comment:
At this point, can we simply refer to these as nodes and assume that they're in a branch path?
```suggestion
  @Input() nodes: any[] = [];
```